po/fr.po

```diff
@@ -131,11 +131,11 @@ msgstr "erreur : option '{}' inconnue dans la section [options]"
 
 #: src/config.rs:896
 msgid "option '{}' does not take a value"
-msgstr "l'option '{}' n'accepte pas de valeur"
+msgstr "l'option '{}' ne prends pas de valeur"
```

Comment:
À la ligne 62 tu as fais la modification inverse. En général pour une traduction, on essaie de garder une certaine consistence. Je pense garder accepte et changer la ligne 62 puisque c'est plus joli

Reply:
La ligne 62 n'utilise pas la meme terminologie en anglais, donc ne veut pas dire précisément la même chose. C'est une question de "phrasing" :)

Reply:
Does not take a value = ne prend pas de valeur

Does not accept a value = n'accepte pas de valeur

Je me dis que s'ils on fait la différence en anglais c'est que ca doit avoir une certaine différence lors de l'utilisation de ces deux phrases

Reply:
J'avais pensé aller voir la traduction de pacman, c'est ce qu'on trouvera de plus proche

Reply:
Il n'y a pas d'équivalence sur pacman : https://gitlab.archlinux.org/pacman/pacman/-/blob/master/src/pacman/po/fr.po
Mon avis serait de conserver la différence dans l'immédiat. Mais [j'ai posé la question](https://github.com/Morganamilo/paru/discussions/433#discussioncomment-971350) sur la discussion à propos des localisations.